### ISSUE TITLE
instructor add session: closing time text box gives an NaaN error on Safari

### ISSUE THREAD

**Issue by damithc** [TEAMMATES/teammates#4950]
Reported by user
![image](https://cloud.githubusercontent.com/assets/1673303/13633164/538566b6-e628-11e5-8d1d-f014c2b7f01e.png)

**Comment by damithc** [comment]
Can someone with a safari browser confirm?

**Comment by wkurniawan07** [comment]
Yup, it's a Safari-specific bug.

**Comment by damithc** [comment]
Ok, someone using safari needs to fix it soon.

**Comment by Gisonrg** [comment]
Not sure why only Safari cannot see it, but here is a possible analysis and suggest fix:
In `datepicker.js`, we have defined:
```
var yesterday = today.setDate(today.getDate() - 1);
var tomorrow = today.setDate(today.getDate() + 1);
```
But `setDate` will actually return a number of milliseconds since 1 January 1970 ([doc](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setDate)), so basically our `yesterday` and `tomorrow` will be two **number** instead of a Date object.
 
Now we pass `tomorrow` as a default date to initialize a datepicker. As the [jQuery UI document](http://api.jqueryui.com/datepicker/#option-defaultDate) suggests, if the `defaultDate` is a number, then it is a number of days from today (-1 means yesterday, +1 means tomorrow, etc.). Since `tomorrow` is a very big number (milliseconds since 1 January 1970), datepicker will try to use the `tomorrow` as a offset from today and create that new date, but it realized the offset is too much and the date is an "invalid date". It will then use a "new Date()" as the displayed date.

Here is a debug screenshot in the jQuery UI source code, you can see the date is a number and newDate is "Invalid Date"
<img width="958" alt="screen shot 2016-03-10 at 00 00 47" src="https://cloud.githubusercontent.com/assets/4332224/13641501/9ce1d6fc-e653-11e5-9354-29fb5875f763.png">

That's why even if on chrome we can see the picker, the default date for `end date` is actually today and not tomorrow :P

So I guess if we either fix `yesterday` or `tomorrow` to make them a valid `Date` object and pass to datepicker, or we simply use the number format (-1 for yesterday, +1 for tomorrow) and pass it to datepicker as default date.
Basically, a simple fix like
```
$("#enddate").datepicker({
    dateFormat: "dd/mm/yy",
    showOtherMonths: true,
    gotoCurrent: true,
    defaultDate: +1
});
```
works great

Not sure if my reasoning is correct, but it would fix the bug :stuck_out_tongue: anyway
